### PR TITLE
@orta: Stub indicator

### PR DESCRIPTION
**Pull request by ashfurrow**
Does a few things:

- Adds an orange "Stubbing" flag when stubbing responses.
- Consolidates reachability testing into one function, which takes stubbing into account. 
- Lets unit tests pass while the testing machine is offline (say, on a plan :airplane:)

I did this all on a flight, so it still needs some polish/testing.

### CODE REVIEW

**Comment by acjay** [comment]
I dig the concept of being able to use as much functionality as possible in isolation of any other systems :+1:

**Comment by ashfurrow** [comment]
Yeah, me too! It kind of just happened – as I reworked the code, I realized "hey, wait a minute..."

File: Kiosk/Images.xcassets/StubbingFlag.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "filename" : "StagingFlag@2x.png",
```
Comment (orta) [inline Kiosk/Images.xcassets/StubbingFlag.imageset/Contents.json]: I think this image should probably look different, give the S for staging - I took a quick stab at this

stubs

Comment (ashfurrow) [inline Kiosk/Images.xcassets/StubbingFlag.imageset/Contents.json]: Agreed, way better :+1:

**Comment by ashfurrow** [comment]
Ran a smoke test on this, removed the WIP.

@orta: :green_apple:

**Comment by orta** [comment]
:+1:  ok, let's ship this

**Pull request merged**